index.js

```diff
@@ -0,0 +1,110 @@
+const addButtonElement = document.querySelector('.add');
+const inputElement = document.querySelector('#task');
+const toDo = document.querySelector('.to-do');
+const doing = document.querySelector('.doing');
+const done = document.querySelector('.done');
+
+let deleteButton;
+let rightArrowButton;
+let deleteButtonDoingSection;
+let rightArrowButtonDoingSection;
+let leftArrowButtonDoingSection;
+let deleteButtonDoneSection;
+let leftArrowButtonDoneSection;
+let newDiv;
```

Comment:
The `newDiv` and `newDivChild` variables are only used in the `addNewToDoAndMoveElement` function and don't need to be declared at the top.